EIPS/eip-3386.md

```diff
@@ -0,0 +1,315 @@
+---
+eip: 3386
+title: ERC-721 and ERC-1155 to ERC-20 Wrapper
+author: Calvin Koder <ashrowz@mailbox.org>
+discussions-to: https://github.com/ethereum/EIPs/issues/3384
+status: WIP
+type: Standards Track
+category: ERC
+created: 2021-03-12
+requires: 165
+---
+
+## Simple Summary
+A standard interface for contracts that create generic ERC-20 tokens which derive from a pool of unique ERC-721/ERC-1155 tokens.
+
+## Abstract
+
+This standard outlines a smart contract interface to wrap identifiable tokens with fungible  tokens. This allows for derivative ERC-20 tokens to be minted by locking the base ERC-721 non-fungible tokens and ERC-1155 multi tokens into a pool. The derivative tokens can be burned to redeem base tokens out of the pool. These derivatives have no reference to the unique id of these base tokens, and should have a proportional rate of exchange with the base tokens. As representatives of the base tokens, these generic derivative tokens can be traded and otherwise utilized according to ERC-20, such that the unique identifier of each base token is irrelevant.
```

Comment:
extra space
```suggestion
This standard outlines a smart contract interface to wrap identifiable tokens with fungible tokens. This allows for derivative ERC-20 tokens to be minted by locking the base ERC-721 non-fungible tokens and ERC-1155 multi tokens into a pool. The derivative tokens can be burned to redeem base tokens out of the pool. These derivatives have no reference to the unique id of these base tokens, and should have a proportional rate of exchange with the base tokens. As representatives of the base tokens, these generic derivative tokens can be traded and otherwise utilized according to ERC-20, such that the unique identifier of each base token is irrelevant.
```